default.json

```diff
@@ -25,5 +25,13 @@
   },
   "baseBranches": ["develop"],
   "branchPrefix": "chore/dependencies/",
-  "rebaseWhen": "behind-base-branch"
+  "rebaseWhen": "behind-base-branch",
+  "packageRules": [
+    {
+      "matchPackagePatterns": [
+        "*"
+      ],
+      "semanticCommitType": "chore"
```

Comment:
Is there an advantage of this approach over https://docs.renovatebot.com/semantic-commits/#changing-the-semantic-commit-type

Reply:
no, the preset results in the same config.

see https://docs.renovatebot.com/presets-default/#semanticcommittypeallarg0